File: algorithms/implementation/cavity-map/cavity-map-java/cavity-map-java/src/TesteCavityMap.java
import static org.junit.Assert.assertEquals;

import java.util.Arrays;
import java.util.List;

import org.junit.Test;

public class TesteCavityMap {

    @Test
    public void testeCaso1() {

        List<String> grade = Arrays.asList("989", "191", "111");
        List<String> esperado = Arrays.asList("989", "1X1", "111");
        List<String> resultado = Resultado.cavityMap(grade);

        assertEquals(resultado, esperado);
    }
    
}

File: algorithms/implementation/cavity-map/cavity-map-javascript/js/testeCavityMap.js
import {cavityMap} from '../js/cavityMap.js';
import assert from 'assert';

describe('cavityMap', function(){

    it('A grade modificada é: 989, 1X1, 111', function() {

        let grade = ['989', '191', '111'];
        let esperado = ['989', '1X1', '111'];

        let resultado = cavityMap(grade);

        assert.deepStrictEqual(resultado, esperado);

    });
});

